src/ansys/dpf/core/operators/geo/elements_volume.py

```diff
@@ -11,13 +11,18 @@
 
 
 class elements_volume(Operator):
-    """Compute the volume of each element of a mesh using default shape
-    functions.
+    """Compute the measure of the Elements (volume for 3D elements, surface
+    for 2D elements or length for 1D elements) using default shape
+    functions, except for polyhedrons.
 
     Parameters
     ----------
     mesh : MeshedRegion
     mesh_scoping : Scoping
+        If not provided, the measure of all elements
+        for the mesh is computed. if
+        provided, the scoping needs to have
+        "elemental" location.
```

Comment:
@MichaelNale @rafacanton 
```suggestion
        If not provided, the measure is computed 
        for each element in the mesh. If
        provided, the scoping needs to have
        "elemental" location.
```